index.html

```diff
@@ -9,7 +9,7 @@
         <span class="letters letters-2 letters-purple">Purple</span>
         <span class="letters letters-3 letters-fuchsia">Fuchsia!</span>
     </h1> -->
-    <Img src="assets/images/gifs/anim.gif" alt="Pink, Purple, Fuchsia!" style="width: 100vw;"> </Img>
+    <img src="assets/images/gifs/anim.gif" alt="Pink, Purple, Fuchsia!" style="margin-top:20px;"> </img>
```

Comment:
In HTML the `<img>` tag has no end tag.

https://www.w3schools.com/tags/tag_img.asp